src/coreclr/vm/classhash.cpp

```diff
@@ -223,17 +223,17 @@ VOID EEClassHashTable::ConstructKeyFromData(PTR_EEClassHashEntry pEntry, // IN
         THROWS;
         WRAPPER(MODE_ANY);
         WRAPPER(GC_TRIGGERS);
-        if (m_bCaseInsensitive) INJECT_FAULT(COMPlusThrowOM();); else WRAPPER(FORBID_FAULT);
+        if (IsCaseInsensitiveTable()) INJECT_FAULT(COMPlusThrowOM();); else WRAPPER(FORBID_FAULT);
         SUPPORTS_DAC;
     }
     CONTRACTL_END;
 
     {
 #ifdef _DEBUG_IMPL
-        _ASSERTE(!(m_bCaseInsensitive && FORBIDGC_LOADER_USE_ENABLED()));
+        _ASSERTE(!(IsCaseInsensitiveTable() && FORBIDGC_LOADER_USE_ENABLED()));
 #endif
 
-        // cqb - If m_bCaseInsensitive is true for the hash table, the bytes in Key will be allocated
+        // cqb - If IsCaseInsensitiveTable() is true for the hash table, the bytes in Key will be allocated
```

Comment:
What is cqb? I don't see that anywhere in here.

Reply:
It isn't there at all. This comment is confusing as it refers to a state in the code which hasn't existed in decades. I've tried to fix up the comment to make some amount of sense, but in general everything to do with the case insensitive table is nonsense.